src/bugs/README.md

```diff
@@ -0,0 +1,6 @@
+# The Great Chapter of Bugs
```

Comment:
How about incorporating a :hankey: emoji somewhere around here?

Reply:
This might be too much :smiley:

Reply:
I mean, the transition from `days or weeks` to  `weeks or months` should be enough